src/global.d.ts

```diff
@@ -60,6 +60,7 @@ declare module 'clientConfig' {
     search?: SearchConfiguration;
     featureEditRoles?: FeatureEditConfiguration;
     wfsLockFeatureEnabled?: boolean;
+    documentationButtonVisible?: boolean;
```

Comment:
Could you please add the newly added parameter to the docs in the [README](https://github.com/terrestris/shogun-gis-client?tab=readme-ov-file#configuration-)?

Reply:
see: https://github.com/terrestris/shogun-gis-client/pull/1740